README.md

```diff
@@ -64,3 +57,28 @@ As a Maker
 So that I can find relevant peeps
 I want to filter on a specific keyword
 ```
+## Outcomes
+
+- Testing and coverage:
+Task has been completed in a TDD manner, feature testing with Capybara and unit testing with RSpec was involved. 
+The coverage of 99.19% was achieved. 
+Additionally, `rubocop` accepts the code as clean.
+
+- Reflection on blockers:
+The biggest blocker I have encountered while working on the project, was making sure to wrap Database data in program objects. It took me some time to understand properly how it works, but having to complete pair-programming challenge for this week along with trying to sovle same problem with `Chitter` has in the end made me deeply understand this initial blocker. 
+Additionally, I have also found some difficutlies in grasping how to write and design `REASTful` paths for my controllers. However, aftyer consulting it with my peers, reading around the topic (resources that helped me: [here](https://github.com/makersacademy/course/blob/master/pills/rest.md),  [here](http://makers-academy.slides.com/makersacademy/rest) and [here](https://github.com/sjmog/rest)) and consulting it with my coach, again, it made me forge my struggle into a good learning activity. 
+
+- Code quality: 
+App controllers were ketp "skinny" and all the logic was kept in dedicated `Peep` class. 
+`HTML` views werre not involved in logic as advised, and I have had a chance to work on fine tunning simple `HTML` and `CSS` edditing. 
+Controllers' paths were designed wit `RESTful` convention. 
+
```

Comment:
I like how you went into detail about your blockers. Also well done on getting the near perfect code coverage!